umbraco-cms/reference/notifications/editormodel-notifications/customizing-the-links-box.md

```diff
@@ -9,14 +9,25 @@ For a content item, Umbraco will show a **Links** box within the **Info** conten
 
 ![image](images/properties-info-app.png)
 
-With the `SendingContentNotification` event, we can manipulate the links in the `Urls` property - e.g. replace it with some custom links (although a URL provider would be more suitable):
+With the `SendingContentNotification` event, we can manipulate the links in the `Urls` property. This could be by replace it with custom links although a URL provider would be more suitable:
 
 ```C#
 public void Handle(SendingContentNotification notification)
 {
     notification.Content.Urls = new[]
     {
-        new UrlInfo($"/products/?id={notification.Content.Id}", true, CultureInfo.CurrentCulture.Name)
+        new UrlInfo($"/products/?id={notification.Content.Id}", true, null)
+    };
+}
+
+It is also possible to specify the culture:
```

Comment:
```suggestion
If the content item has multiple cultures, we can specify the link culture like this: 
```

Reply:
If this rephrase makes more sense, the phrase _"or remove the box entirely by providing an empty list of links:"_ later in this document should also be updated to something like _"We can also remove the Links box entirely by providing an empty list of links:"_

Reply:
Where exactly are you proposing to add this rephrasing? 😇